crates/hc_seed_bundle/src/seed_cipher.rs

```diff
@@ -205,12 +200,12 @@ impl LockedSeedCipherPwHash {
 
 /// This locked cipher is based on security questions.
 pub struct LockedSeedCipherSecurityQuestions {
-    salt: sodoken::BufReadSized<16>,
+    salt: U8Array<16>,
```

Comment:
There are many, many cases where we used to have types that hid where memory was locked or not. I've done my best to determine what's secret and what isn't at the point that data is created. So passphrases are clearly secret, public keys and salts are not.